part_1/title_v/sekcja_31_PL.md

```diff
@@ -0,0 +1,32 @@
+## <a name="title_5"></a><p align="center">T Y T U Ł &emsp; V<br /> LICENCJA &emsp; LUB &emsp; POZWOLENIE &emsp; NA &emsp; BROŃ &emsp; PALNĄ</p>
```

Comment:
W tytule rozdziału po angielsku i po czesku jest mowa tylko o licencjach. Pozwolenia to osobna rzecz, licencje - osobna. Chyba, że masz jakieś tajne info, że tutaj to powinno być właśnie tak?

Reply:
[Tutaj](https://github.com/postscripter8/czeska_ustawa_o_broni/wiki/Nazwy-w%C5%82asne-u%C5%BCywane-w-ustawie-i-jej-t%C5%82umaczenia) są nazwy własne stosowane w tekście (niektóre przeze mnie wymyślone) - nie trzeba wymyślać :P.